saga/saga.go

```diff
@@ -182,17 +182,9 @@ func (s *Saga) updateSagaStateLoop() {
 func (s *Saga) itr(update sagaUpdate) {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
-	currState := s.state
-	newState, err := logMessage(currState, update.msg, s.log)
-
-	if err == nil {
-		s.state = newState
-
-		update.resultCh <- nil
-
-	} else {
-		update.resultCh <- err
-	}
+	var err error
```

Comment:
while we are changing this area of the code, can we change the name to something like 'lockedUpdate'